jsonschema-core.xml

```diff
@@ -532,7 +532,7 @@
                     Keywords MAY use regular expressions to express constraints, or constrain
                     the instance value to be a regular expression.
                     These regular expressions SHOULD be valid according to the regular expression
-                    dialect described in <xref target="ecma262">ECMA 262, section 15.10.1</xref>.
+                    dialect described in <xref target="ecma262">ECMA 262, section 21.2.1</xref>.
```

Comment:
you may want to link to https://tc39.es/ecma262/#sec-patterns or https://www.ecma-international.org/ecma-262/11.0/#sec-patterns section numbers change over time, so even if you're pegging to ES2020, if in the future you wanted to compare to a later edition, using the link anchors will make that easier.

Reply:
@awwright Can you make xrefs link to specific sections of the target? I don't see how we can do this.

Reply:
Looks like we can only do this if we move to using  rfc2629.xslt (https://github.com/reschke/xml2rfc) rather than our current xml2rfc approach. As this is non-minimal work, I'll leave as is for now.